pkg/credentialprovider/config.go

```diff
@@ -282,7 +282,14 @@ func (ident DockerConfigEntry) MarshalJSON() ([]byte, error) {
 // decodeDockerConfigFieldAuth deserializes the "auth" field from dockercfg into a
 // username and a password. The format of the auth field is base64(<username>:<password>).
 func decodeDockerConfigFieldAuth(field string) (username, password string, err error) {
+
+	// encoded base64 field length should be a multiple of 4 if it's correctly padded
+	// if not, let's add the padding character = to reach a multiple of 4
+	if PaddingLength := len(field) % 4; PaddingLength != 0 {
```

Comment:
Nit: In Golang, variables defined within the function follow camel case (i.e. `paddingLength`).

Reply:
Good catch, these are my first Golang lines of code and I didn't know that rule :)